event-adapter/README.md

```diff
@@ -0,0 +1,181 @@
+```
+Now you can see:
+* namespaces/{namespace}/events/{eventName}
+    * GET : it will allow you to retrieve from Stackdriver a single event giving the name {eventName} and namespace {namespace} in which it occurs.
+ * namespaces/{namespace}/events
+    * GET : it will allow you to retrieve from Stackdriver a list of events with the given namespace {namespace} in which them occurs.
+    * POST : it is not implemented yet, it should allow you to store in Stackdriver an event in the given namespace {namespace}.
+ * events
+    * GET : it will retrieve from Stackdriver a list of events.
+
+For example, calling:
+ ```sh
+curl -X GET http://localhost:${KUBE_PORT}/apis/v1events/v1alpha1/namespaces/default/events
+ ```
+you will get something like this:
+```
+{
+  "kind": "EventList",
+  "apiVersion": "v1events/v1alpha1",
+  "metadata": {
+	"selfLink": "/apis/v1events/v1alpha1/namespaces/default/events"
+  },
+  "items": [
+	{
+	  "kind": "Event",
+	  "apiVersion": "v1",
+	  "metadata": {
+		"name": "event-adapter-3051674978-3mt16.14e3920fe82c7b4c",
+		"namespace": "default",
+		"selfLink": "/apis/v1events/v1alpha1/namespaces/default/events",
+		"uid": "ea55efed-9799-11e7-be52-42010a800002",
+		"resourceVersion": "22248",
+		"creationTimestamp": "2017-09-12T09:08:17Z"
+	  },
+	  "involvedObject": {
+		"kind": "Pod",
+		"namespace": "default",
+		"name": "event-adapter-3051674978-3mt16",
+		"uid": "e7756e8d-9799-11e7-be52-42010a800002",
+		"apiVersion": "v1",
+		"resourceVersion": "2290232",
+		"fieldPath": "spec.containers{pod-event-adapter}"
+	  },
+	  "reason": "Pulled",
+	  "message": "Successfully pulled image \"gcr.io/erocchi-gke-dev-1attempt/event-adapter:1.0\"",
+	  "source": {
+		"component": "kubelet",
+		"host": "kubernetes-minion-group-lbfg"
+	  },
+	  "firstTimestamp": "2017-09-12T09:08:17Z",
+	  "lastTimestamp": "2017-09-12T09:08:17Z",
+	  "count": 1,
+	  "type": "Normal"
+	},
+
+	...
+	]
+}
+```
+Other examples may be:
+```sh
+curl -X GET http://localhost:${KUBE_PORT}/apis/v1events/v1alpha1/events
+```
+You can also retrieve a single event, you should set EVENT_NAME variables equal to the name of the event. Then call:
+```sh
+curl -X GET http://localhost:${KUBE_PORT}/apis/v1events/v1alpha1/namespaces/default/events/$EVENT_NAME
+```
+Then, there is the one that is not supported yet.
+```sh
+curl -X POST http://localhost:${KUBE_PORT}/apis/v1events/v1alpha1/namespaces/default/events
+```
+There are flags you can use if you want to limit the events you are going to retrieve.
+* ***max-retrieved-events=N***
+If you want to retrieve at most N events. The default of 100 events will be used.
+* ***retrieve-events-since-millis=T***
+It will retrieve only events that were logged after timestamp T (number of milliseconds after epoch -- 1 January 1970). The default of last 1h will be used.
+If T < 0 the default option will be used as well.
+
+In order to change these flags you can change adapter.yaml file.
```

Comment:
"In order to change these flags you can change adapter.yaml file" -> "In order to change these flags, modify the adapter.yaml file"

Reply:
Done